packages/svelte/src/internal/client/runtime.js

```diff
@@ -128,7 +128,8 @@ export function set_untracked_writes(value) {
 }
 
 /** @type {number} Used by sources and deriveds for handling updates to unowned deriveds it starts from 1 to differentiate between a created effect and a run one for tracing */
```

Comment:
unrelated to this PR but I think this comment is out of date? write versions are used for everything, not just unowned deriveds.

separately, would be good to have a comment explaining what `read_version` is